validation-test/Python/gyb.swift

```diff
@@ -0,0 +1,4 @@
+// RUN: %{python} -m doctest %S/../../utils/gyb.py
+// RUN: echo 'Hello ${ME}' | %S/../../utils/gyb --test -DME=Swift | FileCheck %s
+// RUN: echo 'Hello ${ME}' | %S/../../utils/gyb --verbose-test -DME=Swift | FileCheck %s
```

Comment:
Could you use the `%gyb` substitution?  You can also add a `%gyb-py` one to `test/lit.cfg`.